js/app/modules/card/thumb.js

```diff
@@ -108,65 +25,206 @@ const Thumb = new Module.Class({
     Implements: [Card.Card],
 
     Template: 'resource:///com/endlessm/knowledge/data/widgets/card/thumb.ui',
-    InternalChildren: [ 'thumbnail-frame', 'inner-grid', 'content-frame', 'title-label', 'synopsis-label' ],
+    InternalChildren: [ 'layout', 'text-layout',  'title-label', 'synopsis-label', 'grid', 'thumbnail-frame'],
 
     _init: function (props={}) {
         this.parent(props);
 
+        this.set_thumbnail_frame_from_model(this._thumbnail_frame);
+
         this.set_title_label_from_model(this._title_label);
         this.set_label_or_hide(this._synopsis_label, this.model.synopsis);
+
         this._context_widget = this.create_context_widget_from_model();
-        this._inner_grid.add(this._context_widget);
+        this._context_widget.halign = Gtk.Align.START;
+        this._text_layout.add(this._context_widget);
 
-        this.set_size_request(Card.MinSize.B, Card.MinSize.B);
-        this._layout = new ThumbLayout(this._thumbnail_frame, this._content_frame, {
-            visible: true,
-        });
-        this.add(this._layout);
-        this.set_thumbnail_frame_from_model(this._thumbnail_frame);
+        this.show_all();
```

Comment:
This breaks this._synopsis_label visibility, unless you makae it no-show-all (which we should)
Also it would show the widget even if visible property is false